docs/source/release_notes.rst

```diff
@@ -104,6 +137,26 @@ Requirements
 The IBM z/OS core collection has several dependencies, please review the `z/OS core support matrix`_ to understand both the
 controller and z/OS managed node dependencies.
 
+Known Issues
+------------
+
+- ``zos_job_submit`` - when setting 'location' to 'LOCAL' and not specifying the from and to encoding, the modules defaults are not read leaving the file in its original encoding; explicitly set the encodings instead of relying on the default.
+- ``zos_job_submit`` - when submitting JCL, the response value returned for **byte_count** is incorrect.
+
+- ``zos_job_submit``, ``zos_job_output``, ``zos_operator_action_query`` - encounters UTF-8 decoding errors when interacting with results that contain non-printable UTF-8 characters in the response. This has been addressed in this release and corrected with **ZOAU version 1.2.5.6** or later.
+
+   - If the appropriate level of ZOAU can not be installed, some options are to:
```

Comment:
As well replace this sentence:
```
   - If the appropriate level of ZOAU can not be installed, some options are to:
```
with:
```
   - Some options to work around this known issue are:
```

The rest of the content below , the work around still applies and its indentation and solution is fine.

Reply:
Looks good